modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -9,11 +9,12 @@ Create a vSphere Windows virtual machine (VM) golden image.
 
 .Prerequisites
 
-* You have installed a cluster on vSphere.
+* You have installed a cluster on vSphere configured with hybrid networking using OVN-Kubernetes.
+* You have defined a custom VXLAN port in your hybrid networking configuration to work around the link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere[pod-to-pod connectivity issue between hosts].
 
 .Procedure
 
-. Create the VM from an updated version of the Windows Server 1909 VM image that includes the following link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch].
+. Create the VM from an updated version of the Windows Server 1909 VM image that includes the following link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. This patch is not available for the `Windows Server 2019` VM image.
```

Comment:
By saying "the following patch" here, I'm not sure if it refers to the patch in the link or in a step to follow. If it's the patch in the link, maybe you could say something like the following?
`...that includes the following Microsoft patch: link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[KB4565351]`

Reply:
Or alternatively:
`...that includes the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351].`